app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/SettingsViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package nerd.tuxmobil.fahrplan.congress.settings
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.stateIn
+import kotlinx.coroutines.launch
+import nerd.tuxmobil.fahrplan.congress.preferences.SettingsRepository
+
+internal class SettingsViewModel(
+    private val settingsRepository: SettingsRepository,
```

Comment:
Maybe `settingsRepository` -> `repository`, if no other repository is planned to be added.

Reply:
`AppRepository` will be introduced in a later stage.